openshift-test-utils/src/main/java/io/openshift/booster/test/OpenShiftTestAssistant.java

```diff
@@ -176,4 +176,29 @@ public void awaitPodReadinessOrFail(Predicate<Pod> filter) {
                                                   }
         );
     }
+
+    public void scale(final int replicas) {
+        System.out.println(String.format("Scaling replicas from %s to %s.", deploymentConfig().getStatus()
+                .getAvailableReplicas(), replicas));
+
+        this.client
+                .deploymentConfigs()
+                .inNamespace(this.project)
+                .withName(this.applicationName)
+                .scale(replicas);
+
+        await().atMost(5, TimeUnit.MINUTES)
+                .until(() -> deploymentConfig()
+                        .getStatus()
+                        .getAvailableReplicas() == replicas
```

Comment:
Could we please use the approach that works with OpenShift 3.4? See https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-configmap/commit/271515401cb15973546cb63cb42550696592e118